vision_agent/lmm/lmm.py

```diff
@@ -54,7 +84,7 @@ def __init__(
         self,
         model_name: str = "gpt-4o",
         api_key: Optional[str] = None,
-        max_tokens: int = 1024,
+        max_tokens: int = 4096,
```

Comment:
Nice!! Also i would look into top_p to see if it has an impact. Here is a thread on what top_p is, i like the Hugging face link with visualizations